<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWindow</class>
 <widget class="QWidget" name="DebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="L_DataSet2SizeBytes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L_DataSet2SizeBytes</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_DataSet2SizeOffset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L_DataSet2SizeOffset</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_DataSet1SizeBytes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L_DataSet1SizeBytes</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_DataSet1SizeOffset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L_DataSet1SizeOffset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
